docs/source/release_notes.rst

```diff
@@ -2,19 +2,21 @@
 
 Release Notes
 -------------
-.. **Future Release**
+**Future Release**
     * Enhancements
     * Fixes
     * Changes
     * Documentation Changes
     * Testing Changes
+        * Fix non-deterministic Dask test ``test_calculate_feature_matrix::test_entities_relationships`` (:pr:`1294`)
```

Comment:
Including the exact test name is maybe a bit too specific for a testing entry - if people want that info they can check the PR

Reply:
Updated